tests/testthat/test-package.R

```diff
@@ -109,7 +109,7 @@ test_that("can extract lifecycle badges from description", {
   )
 
   expect_equal(extract_lifecycle(rd_desc), "deprecated")
-  expect_equal(extract_lifecycle(rd_param), NULL)
+  expect_null(extract_lifecycle(rd_param))
```

Comment:
I don't like the use of `expect_null()` in cases like this because I think it makes it harder to see the parallels between the test cases.